mappings/remind_2.1.yaml

```
@@ -2,6 +2,18 @@ model: REMIND 2.1
 native_regions:
   - CAZ: REMIND 2.1|CAZ
   - CHA: REMIND 2.1|CHA
+  - IND: REMIND 2.1|IND
+  - JPN: REMIND 2.1|JPN
+  - LAM: REMIND 2.1|LAM
+  - MEA: REMIND 2.1|MEA
+  - OAS: REMIND 2.1|OAS
+  - REF: REMIND 2.1|REF
+  - SSA: REMIND 2.1|SSA
+  - USA: REMIND 2.1|USA
+  # 12-region version of REMIND
+  - EUR: EU27 & UK
```

If this is a native region of the 12-region version of REMIND, I suggest the following 
```suggestion
  - EUR: REMIND 2.1|EUR
```
and add a mapping in the common-region section
```
 - EU27 & UK:
     - EUR
```

Otherwise, it could be confusing that results from a 12-region model run will only have 11 `REMIND 2.1|*` regions - better to duplicate the information.